db/migrate/20170807145023_create_timelogs.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTimelogs < ActiveRecord::Migration[5.1]
+  def change
+    create_table :timelogs do |t|
+      t.datetime :start_time
+      t.integer :duration, null: false, default: ""
```

Comment:
It is interesting to see in DB, what posgres did for that. We create integer column with default empty string value.